Examples/src/main/java/com/mapbox/vision/examples/BaseActivity.java

```diff
@@ -0,0 +1,93 @@
+package com.mapbox.vision.examples;
+
+import android.content.pm.PackageInfo;
+import android.content.pm.PackageManager;
+import android.os.Build;
+import android.os.Bundle;
+import android.text.Html;
+import android.text.Spanned;
+import android.widget.Toast;
+
+import androidx.annotation.NonNull;
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.core.content.ContextCompat;
+
+import com.mapbox.vision.mobile.core.utils.SystemInfoUtils;
+import com.mapbox.vision.mobile.core.utils.snapdragon.SupportedSnapdragonBoards;
+import com.mapbox.vision.utils.VisionLogger;
+
+public abstract class BaseActivity extends AppCompatActivity {
+
+    private static final String PERMISSION_FOREGROUND_SERVICE = "android.permission.FOREGROUND_SERVICE";
```

Comment:
```suggestion
    private static final String PERMISSION_FOREGROUND_SERVICE = Manifest.permission.html.FOREGROUND_SERVICE;
```

Reply:
such field requires API level 28. current min is 23